### PR TITLE
Glossary style tightening

### PR DESCRIPTION
**Pull request by ericronne**
[**Check it out**](https://federalist.fr.cloud.gov/preview/18f/doi-extractives-data/glossary-slash-filter-styling)

Made a few overdue adjustments. We may leverage this component for the
state-page filter, so it made sense to address this now.

@gemfarmer let’s review together when you get a sec! thx

### CODE REVIEW

File: _sass/components/_glossary.scss [hunk 1]
```diff
@@ -38,18 +39,22 @@ $term-width-max: 220px;
...
   margin: 0;
   max-width: $term-width-max;
-  padding-bottom: $base-padding;
   width: 100%;
 
   @include respond-to(tiny-up) {
     width: $term-width;
+
+  @include heading('para-md');
+  font-weight: 700;
```
Comment (gemfarmer) [inline _sass/components/_glossary.scss]: This should definitely be indented #picky 
Also, the `font-weight` should be moved before the `width` and the `@include heading('para-md')` should be moved before both of them

File: _sass/components/_glossary.scss [hunk 2]
```diff
@@ -38,18 +39,22 @@ $term-width-max: 220px;
...
   }
 }
 
 .glossary-definition {
-  font-family: $base-font-family;
   margin-bottom: $base-padding-lite;
   padding-bottom: $base-padding;
+
+  @include heading('para-md');
```
Comment (gemfarmer) [inline _sass/components/_glossary.scss]: The heading should be the first thing declared in this rule

**Comment by gemfarmer** [comment]
@ericronne Did you want the font to be this large in small screens?

screen shot 2017-04-28 at 11 38 12 am

**Comment by ericronne** [comment]
The chevron jumps a little horizontally on click. We should switch to svg. Will issue.

**Comment by ericronne** [comment]
https://github.com/18F/doi-extractives-data/issues/2284

**Pull request merged**